zerver/views/__init__.py

```diff
@@ -758,6 +759,12 @@ def home(request):
     else:
         notifications_stream = ""
 
+    user_agent = request.META['HTTP_USER_AGENT']
+    suggestMobileForward = False
+    if "iPhone" in user_agent or "iPad" in user_agent:
+        suggestMobileForward = True
+    # TODO: Add check(s) for andriod devices
```

Comment:
typo in the spelling here; but otherwise I think it's fine to leave this off until the Android app supports non-zulip.com servers.